pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
@raisedadead is this okay? You reverted it to 6.1 not long ago, but pnpm add new-package sets it from 6.1 to 6.0

Reply:
Looks like [the renovate PR's are doing the same](https://github.com/freeCodeCamp/freeCodeCamp/commit/e6b88035cf84cbeb3f8537012ff1ccf640dfaa4e#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbR1) - so it's probably fine.

Reply:
I believe the version rolled back somewhere between pnpm version 8.3 and 8.6

Reply:
Thanks, I think we're good then.